### ISSUE TITLE
Radio and check boxes

### ISSUE THREAD

**Issue by gary-dalton** [gary-dalton/bowlsmke-issues#83]
For accessibility, these are not visible enough when not filled. Change the standard unchecked setting to
border-gray-500
